back/app/Http/Controllers/MessageController.php

```diff
@@ -35,6 +35,17 @@ public function update(MessageRequest $request,$message_id)
         return response()->json(MessageResource::make($message), JsonResponse::HTTP_CREATED);
     }
 
+    public function updateActivated(Request $request, $message_id)
+    {
+        $message = Message::withTrashed()->findOrFail($message_id); //Vu qu'il y a un soft delete j'ai mis le withTrashed()
+        $this->authorize('updateActivated', $message);
+        $request->validate(['is_active' => 'required|boolean']);
+        $message->update($request->only('is_active'));
```

Comment:
Il faut que tu indiques à Eloquent précisément le champ que tu veux modifier sur le model

```suggestion
        $message->update(['is_active' => $request->only('is_active')]);
```